Run.command

```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-py="$( which python )"
-py3="$( which python3 )"
+[[ -x "$( command -v python3 )" ]] && python3="true"
```

Comment:
check if python3 is an executable in $PATH, set flag if it is. I don't think we need the actual path guys 🤔 they could have aliased it. Let the shell resolve it.